sql/core/src/test/resources/sql-tests/results/ansi/math.sql.out

```diff
@@ -761,7 +761,7 @@ org.apache.spark.SparkArithmeticException
   "errorClass" : "ARITHMETIC_OVERFLOW",
   "sqlState" : "22003",
   "messageParameters" : {
-    "alternative" : "",
+    "alternative" : " Use 'try_conv' to tolerate overflow and return NULL instead.",
```

Comment:
Fundamental: Why do we need to put a sentence into alternative? Is there a way to put the nglish portion into the message text?